code/obj/machinery/flasher.dm

```diff
@@ -228,13 +229,16 @@
 	return
 
 /obj/machinery/flasher/portable/HasProximity(atom/movable/AM as mob|obj)
```

Comment:
I checked and this will trigger at least three times for someone running past where the old one would mostly do the simpler checks then go on cooldown due to flashing.

Could store the person and last checked and then clean it up after a time period but I'm not sure if that is any less expensive.

Reply:
Could have a successful id scan update `last_flash` a bit to let them pass.  Would allow for dragging folks through and crafty folks to piggyback behind sec going through them.

Reply:
I'm using 1.5 seconds which is slightly longer (like 0.4) than it takes for a sec to run through all three tiles so there is a slight window for following.

I considered using our cooldown helpers but last_flash is referenced in other code so it seems better not to mess with it.